mllib/src/main/scala/org/apache/spark/ml/feature/StandardScaler.scala

```diff
@@ -29,7 +29,18 @@ import org.apache.spark.sql.types.{StructField, StructType}
 /**
  * Params for [[StandardScaler]] and [[StandardScalerModel]].
  */
-private[feature] trait StandardScalerParams extends Params with HasInputCol with HasOutputCol
+private[feature] trait StandardScalerParams extends Params with HasInputCol with HasOutputCol {
+  val withMean: BooleanParam = new BooleanParam(this, 
+    "withMean", 
+    "Center data with mean before scaling"
```

Comment:
style: No need to do one parameter per line, except in method and class definitions.  In a method call like this, please consolidate to fewer lines.